draft-ietf-httpbis-origin-frame.md

```diff
@@ -153,6 +153,16 @@ The Origin Set is also affected by the 421 (Misdirected Request) response status
 MUST create the ASCII serialisation of the corresponding request's origin (as per {{!RFC6454}},
 Section 6.2) and remove it from the connection's Origin Set, if present.
 
+Note:
+
+: A server that acts as an alternative service {{?RFC7838}} might remove the origin that a
+  client creates the connection for by sending an ORIGIN frame.  The single value that is in the
```

Comment:
It doesn't remove it from the origin set, that's unfortunate phrasing.  It does invalidate the connection for use with the origin.